### PR TITLE
Reporting service: Creates kafka consumer and plain django REST endpoints

### PR DESCRIPTION
**Pull request by srikantviswanath**
* kafka consumer in a separate process listening to `order.created` topic, that updates `Order` django model
* Exposes a simple detail endpoint reading `/orders/<id>`

### CODE REVIEW

**Comment by giveredindia** [comment]
I think we should move the `reporting_service` out of the spike

**Comment by sbiyyala** [comment]
I think we can move the `reporting_service` out of the spike

File: spike/consumer.py [hunk 1]
```diff
@@ -13,17 +13,15 @@ def consumer_ready(consumer):
...
+                             auto_offset_reset='earliest',
+                             bootstrap_servers=['localhost:9092'])
+    # consumer_ready(consumer)
+    # loop and print messages
+    for msg in consumer:
+        print(msg)
 
 
 if __name__ == '__main__':
-    exit(main(sys.argv))
```
Comment (sbiyyala) [inline spike/consumer.py:29]: Was `exit` causing any issues?

**Review by sbiyyala — COMMENTED** [review]
Generally looks good; we should proceed with merge once `order-service` is moved out of spike

File: spike/consumer.py (hunk 1 above)
Comment (srikantviswanath) [reply spike/consumer.py:29]: actually no, I was wondering why we put it in the first place.
So tried to remove and test, forgot to put it back

Comment (sbiyyala) [reply spike/consumer.py:29]: `sys.exit(main(sys.argv))` will trigger the `sys.exit` function with an exit code, equivalent to running the main function with the received `command line` arguments. 
A reasonable practice for anything 'executable'

Comment (srikantviswanath) [reply spike/consumer.py:29]: Yup agree, my experience with python is that this is the implicit behavior by default

Comment (sbiyyala) [reply spike/consumer.py:29]: Great - please merge once this change is reverted

**Pull request merged**